.babelrc

```diff
@@ -1,9 +1,14 @@
 {
   "plugins": [
-    "transform-object-rest-spread",
-    "transform-object-assign",
-    "transform-class-properties",
+    ["@babel/plugin-transform-flow-strip-types", {"requireDirective": true}],
```

Comment:
Babel seems to barf up on files like [PrefixIntervalTree.js](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/vendor_upstream/struct/PrefixIntervalTree.js), due to flow annotations.
Using this plugin in to strip them off.